src/libraries/System.Text.Json/tests/System.Text.Json.TrimmingTests/System.Text.Json.TrimmingTests.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This is not following the existing trimming test pattern.

See the files in https://github.com/dotnet/runtime/tree/main/src/libraries/System.Text.Json/tests/System.Text.Json.Tests/TrimmingTests

1. Add a `MyTest.cs` file to that folder with a `static int Main` entry point that exercises the code you want.
2. Add that file as a `TestConsoleAppSourceFiles` entry in https://github.com/dotnet/runtime/blob/main/src/libraries/System.Text.Json/tests/System.Text.Json.Tests/TrimmingTests/System.Text.Json.TrimmingTests.proj

Reply:
Is it intentional that the existing `TrimmingTests` project is not included in the `sln` and is a nested folder inside the `System.Text.Json.Tests` folder? If not, I'd be inclined to move the existing tests to the folder that this PR is adding.

Reply:
It is not intentional for the `TrimmingTests` folder to be nested inside the `System.Text.Json.Tests` folder. Typically we split the `TrimmingTests` folder in a separate folder from the unit tests. But some libraries didn't take the time to move the existing unit tests down a folder from the top-level `tests` folder, which is what happen for JSON in https://github.com/dotnet/runtime/pull/38458/. But now that the tests folder is moved split apart (since https://github.com/dotnet/runtime/pull/51300), the `TrimmingTests` folder being directly under `src/libraries/System.Text.Json/tests` would make more sense.

See:

* https://github.com/dotnet/runtime/tree/main/src/libraries/Microsoft.Extensions.Http/tests
* https://github.com/dotnet/runtime/tree/main/src/libraries/Microsoft.Extensions.Hosting/tests
* https://github.com/dotnet/runtime/tree/main/src/libraries/System.Net.Http/tests

for "typical" examples of having the `tests` folder split apart.